src/containers/home-container/HomeContainer.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import ReactModal from 'react-modal'
+
+import HomePage from '../../pageComponents/HomePage'
+
+export default class HomeContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      movies: [],
+      showMovieDetailsModal: false,
+    };
+  }
+
+  async componentDidMount() {
+      try {
+          const responsePromise = await fetch(`https://api.themoviedb.org/3/trending/movie/day?page=1&api_key=${process.env.REACT_APP_API_KEY}`, {
```

Comment:
I was thinking we should move the base URL `https://api.themoviedb.org` into a constant because it's an easy mistake to misspell when the students are pairing